### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: Unable to access the links present under plugin related information in Browse repositories dialog. 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1454]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. IntelliJ IDEA start screen appears, Navigate and select  Plugins under Configure combo box.
3. Plugins dialog will open.
4. Navigate to " Browse repositories" button and hit enter.
5. In Browse repositories dialog Select any particular plugin(Ex: Azure toolkit for IntelliJ).
6. After selecting the plugin navigate to plugin information using "Ctrl+tab" and try to access the links under Plugin information with keyboard.

**Expected Result:**
User should be able to access the "Links" under Plugin related information using keyboard.

**Actual Result:**
"Links" under Plugin related information is not accessible with keyboard.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version**: Version 1709(Os Build 16299.192)

![links are not accessible in browse repositories](https://user-images.githubusercontent.com/22956130/35800800-e916be48-0a1e-11e8-89f4-b48749845949.png)

**Comment by Eskibear** [comment]
The dialog is IntelliJ IDEA's own setting page, it's out of the toolkit's scope.

**Comment by Eskibear** [comment]
tracked in https://youtrack.jetbrains.com/issue/IDEA-191938

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. This is the feature of IntelliJ IDEA itself. It's out of the plugin scope.
* **Quality of experience**: not impacted. This cannot be controlled by the plugin since it is the IntelliJ IDEA's feature. But if the user do want to see the introduction of any IntelliJ plugins, he can view the online docs for IntelliJ. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by jdneo** [comment]
Currently, we will only have links that pointing to an issue to our repo. So if the user wants to check the link content, he can use his browser, append the number to the issues link and see the details:

For example:
```
https://github.com/Microsoft/azure-tools-for-java/issues/{number}
```

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749886
